app/assets/stylesheets/base/_text_content.scss

```
@@ -1,10 +1,5 @@
 @mixin margin($margin: 1) {
   margin-bottom: #{$margin}rem;
-  margin-top: #{$margin}rem;
-
-  &:first-child {
-    margin-top: 0;
-  }
 }
 
 @mixin body {
```

hi @dannymidnight here's a suggestion for margins. I think it's beneficial to have top and bottom margins for `h1–h6` and `p` so they'll always create a gap above them for any element (such as `table`, custom `div`s etc).

```suggestion
@mixin margin($margin: 1) {
  margin-bottom: #{$margin}rem;
  margin-top: #{$margin}rem;
}

@mixin body($margin: 1) {
```

---

Makes sense! I've pushed up something similar that brings back the top margins and simplifies usage of the sass mixins. Hopefully this does the trick :)

https://github.com/buildkite/docs/pull/1893/commits/f8a69c2c3a296a934e3bebf8969e2d7f4792f0c0